pinot-controller/src/main/java/org/apache/pinot/controller/BaseControllerStarter.java

```diff
@@ -501,6 +494,28 @@ protected void configure() {
 
     LOGGER.info("Starting controller admin application on: {}", ListenerConfigUtil.toString(_listenerConfigs));
     _adminApp.start(_listenerConfigs);
+    List<String> existingHlcTables = new ArrayList<>();
+    _helixResourceManager.getAllRealtimeTables().forEach(rt -> {
+      TableConfig tableConfig = _helixResourceManager.getTableConfig(rt);
+      if (tableConfig != null) {
+        Map<String, String> streamConfigMap = IngestionConfigUtils.getStreamConfigMap(tableConfig);
```

Comment:
Please follow the same logic as `StreamConfig`, or just use `StreamConfig` to extract the `ConsumerType`

Reply:
In the past, I have been recommended to use `IngestionConfigUtils.getStreamConfigMap` which searches for configs in 2 places - TableConfig#IngestionConfig and TableConfig#IndexingConfig. Likely due for compatibility reasons. 
We should fix that separately (if not already) - re-write existing table configs to move stream configs from indexConfig into IngestionConfig.